uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -829,7 +856,13 @@ define([
       function updateRequest() {
         vm.leaveRequest.update()
           .then(function () {
-            postSubmit('LeaveRequest::updatedByManager');
+            if (vm.role == 'manager') {
+              postSubmit('LeaveRequest::updatedByManager');
```

Comment:
What's the difference between "updatedByManager" and "edit"? Do we need two different events for the same edit functionality?